.maint/requirements.txt

```
@@ -0,0 +1,2 @@
+click
+fuzzywuzzy
```

I'm fine with either approach, but we already have an extras for maintainers.
https://github.com/nipreps/fmriprep/blob/fcd825c30b225941231e1fd4f7d231356a456121/pyproject.toml#L77-L80

In the past I found the speedup to be useful
```suggestion
fuzzywuzzy[speedup]
```

---

This should be updated in https://github.com/nipreps/GOVERNANCE/blob/main/project-docs/.maint/requirements.txt